projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/storage/File_Storage.php

```diff
@@ -83,7 +119,7 @@ public function garbage_collect() {
 			return false;
 		}
 
-		$count = Filesystem_Utils::delete_expired_files( $this->root_path, JETPACK_BOOST_CACHE_DURATION );
+		$count = Filesystem_Utils::gc_expired_files( $this->root_path, JETPACK_BOOST_CACHE_DURATION, Filesystem_Utils::REBUILD );
```

Comment:
Looks like we aren't using `DELETE` for garbage collection at the moment, can we remove the logic related to deleting?

I am also concerned about accidentally left-over files. They will never be deleted now as we are only rebuilding. Perhaps we should remove rebuild files if they are too old during garbage collection?

Reply:
The Logger uses this function to delete old files, so we'll need the delete logic, but I agree about deleting old rebuild files. The TTL for rebuild files should probably be much smaller than the regular TTL. Perhaps 10 seconds, and that should be configurable by a constant.